content/features.yaml

```diff
@@ -1,20 +1,16 @@
--
-  name: design
-  title: Simple, Clean and Modern Design.
-  body: Lorem ipsum dolor sit amet, ea eum labitur scripserit, illum complectitur deterruisset at pro. Odio quaeque reformidans est eu, expetendis intellegebat has ut, viderer invenire ut his. Has molestie percipit an. Falli volumus efficiantur sed id, ad vel noster propriae. Ius ut etiam vivendo, graeci iudicabit constituto at mea. No soleat fabulas prodesset vel, ut quo solum dicunt. Nec et amet vidisse mentitum. Cibo mutat nulla ei eam.
+- name: interactive
+  title: Interactive, collaborative learning spaces
+  body: Encapsulated learning activities in Activities or Inquiry Learning Spaces (ILS) and can be composed of various elements such as text, images, videos, and even more complex and specific technologies such as apps or labs. All these elements are finally presented in a dynamic and attractive layout and can be shared with a set of students or a wider audience.
```

Comment:
"Create encapsulated learning activities composed of text, images, videos, and even apps or labs. All these elements are presented in a dynamic and attractive layout, and can be shared with a specific group of students or with a wider audience."